prae/tests/unchecked.rs

```diff
@@ -0,0 +1,54 @@
+use prae;
+
+prae::define! {
+    pub Username: String
+    ensure |u| !u.is_empty()
+}
+
+#[test]
+#[cfg(feature = "unchecked-access")]
+fn construction_fails_for_invalid_data_unchecked_succeeds() {
+    use assert_matches::assert_matches;
+    use prae::UncheckedAccess;
+
+    assert_matches!(
+        Username::new("").unwrap_err(),
+        prae::ValidationError { .. }
```

Comment:
Oops, there is no `ValidationError` anymore!

Reply:
I've merge in upstream master, but the build is failing due to the serde_integration mod being compiled even when the serde feature is disabled. See pull request  #19 .